plugins/woocommerce/tests/e2e-pw/utils/settings.js

```diff
@@ -15,7 +17,7 @@ function resolvePath( path ) {
  * @return {Promise<void>} A promise that resolves when the update is complete.
  */
 export async function updateValue( path, desiredValue ) {
```

Comment:
(not related to this PR) why are we doing things like this?  Replace one line of code with another line of code. Seems like needless abstraction.